### ISSUE TITLE
Creating a note also creates a POI

### ISSUE THREAD

**Issue by contrapunctus-1** [organicmaps/organicmaps#7620]
⚠ Have you searched for similar, already existing issues?

Yes...all I found were a lot of seemingly-irrelevant issues.

**Describe the issue**
Creating a note using Organic Maps also creates a POI.

**Steps to reproduce**
1. Tap the menu button - Add Place to OpenStreetMap - confirm a location with ✔️ - select a POI type, such as ATM
2. Fill out the box in "Send a note to OSM editors" and hit the ✔️

**Expected behaviour**
A note is created. No POI is created.

**Actual behaviour**
A note is created along with a POI of the type you selected.

e.g. Organic Maps created this POI -
https://www.openstreetmap.org/node/11738355769
...when all I meant to add was this note -
https://www.openstreetmap.org/note/4156476

(This was to demonstrate to a mapping party how to create a note.)

**System information:**
 - LineageOS 20/Android 13
 - Organic Maps version: 2024.03.05-4-FDroid
 - Device Model: OnePlus 9 Pro (LE2121)

**Additional context**
It is not clear to me why a user has to select a POI type in order to create a note. The best solution I can think of right now is - 

1. Show a "Send a note to OpenStreetMap editors" entry under the "Add Place to OpenStreetMap" menu item.  
   AND

2. If the user searches for a POI type and no results are found, OR has been on the POI type screen for more than a few seconds, Organic Maps should offer, "Can't find what you're looking for? Send a note to the OpenStreetMap editors."
   * Or perhaps this item can be shown unconditionally, at the top of the list, at all times. That can also remove the need for the previous suggestion, and thus avoid adding an extra menu item.

While we're at it, the POI type screen doesn't clarify what it's for. Something like "What do you want to add?" or "What are you trying to add?" would be good.

**Comment by biodranik** [comment]
If you don't want to add a POI to OSM, then don't press "Add a place" button.

Long-tap or select any existing object and press Edit Place.

**Comment by contrapunctus-1** [comment]
@biodranik I see. In that case, would you consider adding the ability to open a "standalone" note (i.e. one not associated with an object, for the purpose of marking features one cannot add from Organic Maps) with the flow I've described?

**Comment by biodranik** [comment]
Did you try to long-tap on an empty space and add a note?

**Comment by contrapunctus-1** [comment]
@biodranik Right, I tested that, and it certainly creates a standalone note. Thanks for the tip.

However, I'd still recommend implementing my suggestion, for the following reasons -

1. This difference in behaviour (between writing a note via "Add a place" and writing a note via long press in empty area) is both obscure and inconsistent, for no good reason I can think of.

2. It is often the case that a user is unable to find what they want to add in the POI type screen. At that point, they should be given the option to create a note _instead_ of a POI.

**Comment by biodranik** [comment]
How exactly are you proposing to change the existing functionality to make it more obvious for non-OSM experts (e.g. your grandparents)?

We can even recommend other editors or add something directly at OSM if user can't find that type in OM. Any ideas on the UI?

**Comment by contrapunctus-1** [comment]
@biodranik I made a mockup. When a user selects "Add a place to the map", instead of showing this -
![Screenshot_20240318-201736_Trebuchet half](https://github.com/organicmaps/organicmaps/assets/8513118/2aa5969d-c539-43e0-8642-37a8ef0f3ce5)

I suggest showing something like this -
![proposed note inteface Screenshot_20240318-201736_Trebuchet half](https://github.com/organicmaps/organicmaps/assets/8513118/3a9271ad-9b15-4e5a-9032-ae5ab2954a15)

The text and link should remain visible during preset search, regardless of whether there are any search results.

This way, there is at least a clear and obvious way to create notes without creating POIs (unlike the long-tap-on-empty-area - Edit place - fill out note field behavior, which is not discoverable), and there is a clear alternative path people can take when they cannot find a preset.

To resolve the issue of inconsistency, the "long-press on empty area - swipe up - Edit place" flow for adding a note may actually be removed. The "Add place to OpenStreetMap" flow would be the more discoverable way to create notes (because there's a menu item for it), and it has a step to confirm the location (allowing finer adjustment; finely-positioned long-presses can be a challenge for lay users).

Organic Maps could also clarify that filling out the notes field after selecting a preset results in creating a note _in addition to_ creating the feature.

**Comment by biodranik** [comment]
How ~your grandparents~ most of our users who are not familiar with OSM understand:
1. What is OpenStreetMap?
2. What is a note?
3. Who are those editors?
4. What does a "feature" mean in this context?

Is a better wording or a better UX possible?

**Comment by contrapunctus-1** [comment]
@biodranik That's a bigger issue, which should probably be tackled separately. With "OpenStreetMap", "note", and "OSM editors", I'm just reusing the language already used in Organic Maps. Organic Maps also does a decent job (IMO) of informing the user about OSM.

At best, we can tackle the introduction of "feature", by changing the text to something like -

> Select the kind of object you're trying to add.
> 
> Can't find it here? [Write a note]() for the OpenStreetMap editors instead.

("Object" is already used in the location adjustment screen. "Place" can work, too.)

**Comment by map-per** [comment]
Related to #266

**Comment by raqollu** [comment]
@biodranik 
Does this ui look good on iOS, if yes, i will proceed with the implementation


| Potrait | Landscape |
|--------|--------|
| <img width="1179" height="2556" alt="Image" src="https://github.com/user-attachments/assets/4a07e504-c7db-456f-a7b7-67f7b3a6b693" />| <img width="2556" height="1179" alt="Image" src="https://github.com/user-attachments/assets/cd921099-7ffa-457d-90ab-54d69ce07d83" />|

**Comment by biodranik** [comment]
Thanks! Does it make sense to reuse the layout similar to Android one? Bottom text is a hint inside TextView:

<img width="1280" height="2856" alt="Image" src="https://github.com/user-attachments/assets/d30068dc-7b0d-4a0a-a4bb-d73ef54b6a89" />
